### PR TITLE
Rename gruntwork-cli to gocommons

### PR DESCRIPTION
**Pull request by yorinasub17**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -5,7 +5,7 @@ go 1.14
 require (
 	github.com/aws/aws-sdk-go v1.34.27
 	github.com/blang/semver/v4 v4.0.0
-	github.com/gruntwork-io/gruntwork-cli v0.7.0
+	github.com/gruntwork-io/go-commons v0.8.0
```
Comment (bwhaley) [inline go.mod:8]:
```suggestion
	github.com/gruntwork-io/go-commons v0.8.1
```

**Review by bwhaley — DISMISSED** [review]
LGTM! Might want to bump to 0.8.1, but that's up to you.

**Review by bwhaley — APPROVED** [review]

**Comment by yorinasub17** [comment]
Good idea! Accepted suggestion, and then ran `go mod tidy` 83d076f. Sorry for the toil, but can I get another approval? Thanks!

**Review by zackproser — APPROVED** [review]
LGTM

**Comment by yorinasub17** [comment]
Thanks for review, Merging now!

**Pull request merged**